EIPS/eip-9174.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 9174
+title: On-chain upgrade signaling
```

Comment:
```suggestion
title: Onchain upgrade signaling
```
I am not an editor.  My personal preference is onchain.

Reply:
Well noted.

At this point, on-chain is the preferred usage across other publications in this repo.

I'm happy to adopt the less common usage if we can point to it as a best practice.

And either way that should be after merging this as draft.